src/utils/plot_recipes.jl

```diff
@@ -0,0 +1,117 @@
+
+
+@recipe function StackedPlot(res::StackedArea, variable::String) 
```

Comment:
These recipes should internally call the get_data functions and not requires to functions to see the results. I see the value in having functions that retrieve results in a particular format to use other plotting tools but these ones should be in a single call.